DxOper.pas

```diff
@@ -141,7 +144,7 @@ function TDxOperator.IsMyCall: TCallCheckResult;
 
       D := M[x-1,y-1];
       //'?' matches any char
-      if not (C[x] in [C0[y], '?']) then Inc(D, W_D);
+      if not CharInSet(C[x], [C0[y], '?']) then Inc(D, W_D); //grg1

```

Comment:
"in" works only for AnsiChar, not for Unicode char. Declare C and CO as AnsiString to fix this

On 2015-11-28 13:42, Gerd - DF9TS wrote:
> In DxOper.pas <https://github.com/VE3NEA/MorseRunner/pull/2#discussion_r46083013>:
>
>> @@ -141,7 +144,7 @@ function TDxOperator.IsMyCall: TCallCheckResult;
>>
>>        D := M[x-1,y-1];
>>        //'?' matches any char
>> -      if not (C[x] in [C0[y], '?']) then Inc(D, W_D);
>> +      if not CharInSet(C[x], [C0[y], '?']) then Inc(D, W_D); //grg1
>
> Delphi 10 creates warning for "in", warning suggests usage of CharInSet instead.
> So I replaced "in" with CharInSet() in all places, not just here
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/VE3NEA/MorseRunner/pull/2/files#r46083013>.
>